db.json

```diff
@@ -1,3 +1,186 @@
+      "eventPhase": null,
+      "bubbles": null,
+      "cancelable": null,
+      "defaultPrevented": null,
+      "isTrusted": null,
+      "screenX": null,
+      "screenY": null,
+      "clientX": null,
+      "clientY": null,
+      "ctrlKey": null,
+      "shiftKey": null,
+      "altKey": null,
+      "metaKey": null,
+      "buttons": null,
+      "_id": 3
+    },
+    {
+      "dispatchConfig": null,
+      "_targetInst": null,
+      "isDefaultPrevented": null,
+      "isPropagationStopped": null,
+      "_dispatchListeners": null,
+      "_dispatchInstances": null,
+      "nativeEvent": null,
+      "type": null,
+      "target": null,
+      "eventPhase": null,
+      "bubbles": null,
+      "cancelable": null,
+      "defaultPrevented": null,
+      "isTrusted": null,
+      "screenX": null,
+      "screenY": null,
+      "clientX": null,
+      "clientY": null,
+      "ctrlKey": null,
+      "shiftKey": null,
+      "altKey": null,
+      "metaKey": null,
+      "buttons": null,
+      "_id": 4
+    },
+    {
+      "dispatchConfig": null,
+      "_targetInst": null,
+      "isDefaultPrevented": null,
+      "isPropagationStopped": null,
+      "_dispatchListeners": null,
+      "_dispatchInstances": null,
+      "nativeEvent": null,
+      "type": null,
+      "target": null,
+      "eventPhase": null,
+      "bubbles": null,
+      "cancelable": null,
+      "defaultPrevented": null,
+      "isTrusted": null,
+      "screenX": null,
+      "screenY": null,
+      "clientX": null,
+      "clientY": null,
+      "ctrlKey": null,
+      "shiftKey": null,
+      "altKey": null,
+      "metaKey": null,
+      "buttons": null,
+      "_id": 5
+    },
+    {
+      "dispatchConfig": null,
+      "_targetInst": null,
+      "isDefaultPrevented": null,
+      "isPropagationStopped": null,
+      "_dispatchListeners": null,
+      "_dispatchInstances": null,
+      "nativeEvent": null,
+      "type": null,
+      "target": null,
+      "eventPhase": null,
+      "bubbles": null,
+      "cancelable": null,
+      "defaultPrevented": null,
+      "isTrusted": null,
+      "screenX": null,
+      "screenY": null,
+      "clientX": null,
+      "clientY": null,
+      "ctrlKey": null,
+      "shiftKey": null,
+      "altKey": null,
+      "metaKey": null,
+      "buttons": null,
+      "_id": 6
+    },
+    {
+      "dispatchConfig": null,
+      "_targetInst": null,
+      "isDefaultPrevented": null,
+      "isPropagationStopped": null,
+      "_dispatchListeners": null,
+      "_dispatchInstances": null,
+      "nativeEvent": null,
+      "type": null,
+      "target": null,
+      "eventPhase": null,
+      "bubbles": null,
+      "cancelable": null,
+      "defaultPrevented": null,
+      "isTrusted": null,
+      "screenX": null,
+      "screenY": null,
+      "clientX": null,
+      "clientY": null,
+      "ctrlKey": null,
+      "shiftKey": null,
+      "altKey": null,
+      "metaKey": null,
+      "buttons": null,
+      "_id": 7
```

Comment:
Looks like you are persisting the event object instead of the movie object to the database.